src/Makefile.am

```diff
@@ -707,6 +707,9 @@ uninstall-local: squid.conf.default
 	@$(SHELL) $(top_srcdir)/scripts/remove-cfg.sh "$(RM)" $(DESTDIR)$(DEFAULT_MIME_TABLE) $(srcdir)/mime.conf.default
 	@$(SHELL) $(top_srcdir)/scripts/remove-cfg.sh "$(RM)" $(DESTDIR)$(DEFAULT_CONFIG_FILE) squid.conf.default
 
+distclean-local:
```

Comment:
FWIW, I could not quickly figure out whether .dSYM should be deleted during "make clean" or "make distclean". Are you sure that "make clean" is not the right target for this?

Reply:
AIUI, if an object is generated by make `clean` should erase it. `distclean` is for the extra bits that can be re-generated by another build tool, eg re-running `configure`.

Reply:
yes, clean-local works. Updating